ci/ansible/roles/pulp-restore-backup/defaults/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# location of backup on remote
+remote_bkup_dir: "/root/backups"
+local_bkup_dir: "backups"
+files_to_restore:
+  etc_pulp:
+    dest_dir: /etc/pulp
+    file: etc_pulp
+    archive: etc_pulp.tar.bz2
+  etc_pki_pulp:
+    dest_dir: /etc/pki/pulp
+    file: etc_pki_pulp
+    archive: etc_pki_pulp.tar.bz2
+  var_lib_pulp:
+    dest_dir: /var/lib/pulp
+    file: var_lib_pulp
+    archive: var_lib_pulp.tar.bz2
+db_to_restore:
+  mongodb:
```

Comment:
Same question here as for `db_to_bkup`. Is the extra level of indirection necessary? Or could this variable be written as:

```yaml
db_to_restore:
  archive: mongodb_backup.bz2
  dir: mongodb_backup_dir
```